docs/api/sql/Raster-aggregate-function.md

```diff
@@ -1,9 +1,9 @@
 ## RS_Union_Aggr
```

Comment:
Can you provide some examples of using this function to stack rasters, using at least the example you are discussing in slack for two RGB rasters? I'm unclear on how the index column controls stacking.